samples/star-wars-api/appsettings.Development.json

```diff
@@ -0,0 +1,8 @@
+﻿{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Trace",
+      "Microsoft.Hosting": "Trace" // Debug:Microsoft.Hosting category.
```

Comment:
Do we need this comment?

Reply:
Nope; not sure how that snuck in there.  Removed it in 27b3738f8e333a10d8403a5067eef9ead05c88ad .